chainer/__init__.py

```diff
@@ -147,7 +147,9 @@ def is_arrays_compatible(arrays):
 global_config.use_cudnn = os.environ.get('CHAINER_USE_CUDNN', 'auto')
 global_config.use_cudnn_tensor_core = 'auto'
 global_config.autotune = False
-global_config.use_ideep = os.environ.get('CHAINER_USE_IDEEP', 'never')
+global_config.use_ideep = os.environ.get('CHAINER_USE_IDEEP', 'auto')
```

Comment:
Please revert to `'never'`.

Reply:
Sorry for the mistake and thank you for the review. I'll revert it.